en/docs/get-started/identity-anti-patterns-and-the-identity-bus.md

```diff
@@ -0,0 +1,110 @@
+# Identity Anti-patterns and the Identity Bus
```

Comment:
This doc is not needed in the `get started` section. It should be there in `concepts`